packages/devops/scripts/lambda/resources/startupTupaia.sh

```
@@ -40,23 +40,23 @@ mkdir -m 777 -p $LOGS_DIR
-# if [[ $DEPLOYMENT_NAME == "production" || $DEPLOYMENT_NAME == "dev" ]]; then
+# if [[ $DEPLOYMENT_NAME = production || $DEPLOYMENT_NAME = dev ]]; then
 #     $DEPLOYMENT_SCRIPTS/startCloudwatchAgent.sh |& while IFS= read -r line; do printf '\%s \%s\n' "$(date)" "$line"; done  >> $LOGS_DIR/deployment_log.txt
 # fi
 
 # Add preaggregation cron job if production
-if [[ $DEPLOYMENT_NAME == "production" ]]; then
+if [[ $DEPLOYMENT_NAME = production ]]; then
   \. "$HOME_DIR/.nvm/nvm.sh" # Load nvm so node is available on $PATH
   sudo -u ubuntu echo "10 13 * * * PATH=$PATH $HOME_DIR/tupaia/packages/web-config-server/run_preaggregation.sh | while IFS= read -r line; do printf '\%s \%s\\n' \"\$(date)\" \"\$line\"; done > $LOGS_DIR/preaggregation.txt" > tmp.cron
-  sudo -u ubuntu crontab -l >> tmp.cron || echo "" >> tmp.cron
+  sudo -u ubuntu crontab -l >> tmp.cron || echo >> tmp.cron
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use .. | sudo tee -a file

(SC2024)